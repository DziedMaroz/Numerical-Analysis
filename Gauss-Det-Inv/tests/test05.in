4
0 3 2 1 
3 0 5 6
2 4 0 5
1 7 9 0
