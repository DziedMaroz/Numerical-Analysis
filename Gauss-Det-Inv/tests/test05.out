RUNNING ON TEST: tests/test05.in
det (A) = -242.000
det(A)*A^-1:
  391.00   29.00 -113.00 -103.00
    2.00   28.00  -34.00  -16.00
  -45.00  -25.00   39.00   -3.00
 -158.00  -34.00   24.00   54.00
